### PR TITLE
Updated all CIs

### PR DESCRIPTION
**Pull request by yitam**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ environment:
         TEST_PHP_SQL_SERVER: (local)\SQL2017
         SQL_INSTANCE: SQL2017
         PHP_VC: 15
-        PHP_MAJOR_VER: 7.2
+        PHP_MAJOR_VER: 7.3
         PHP_MINOR_VER: 11
         PHP_EXE_PATH: x64\Release_TS
         THREAD: ts
```
Comment (david-puglielli) [inline appveyor.yml]: PHP_MAJOR_VER below is still 7.1?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -60,7 +60,7 @@ jobs:
```
Comment (david-puglielli) [inline azure-pipelines.yml]: Below we install ODBC 17.3 - should this be updated?

File: appveyor.yml (hunk 1 above)
Comment (yitam) [reply appveyor.yml]: Yes, for now, until after 7.4 is released. In that case we will change the matrix to drop the image VS 2015

File: azure-pipelines.yml (hunk 2 above)
Comment (yitam) [reply azure-pipelines.yml]: Did you mean Windows? Not for now because the job is only a placeholder for future development (see the condition is `false`)

**Pull request merged**